requirements/classes/ccv_bootcamp/requirements.jl

```diff
@@ -1,11 +1,11 @@
 const julia_packages = [
-                    "IJulia", # DO NOT REMOVE
+                   "IJulia", # DO NOT REMOVE
                    # add class specific classes here:
-                   "CSV",
-                   "DataFrames",
-                   "DataFramesMeta",
+                   #"CSV",
```

Comment:
Is Julia going to be used? DataFrames and CSV are pretty basic packages that are often needed.